### DISCUSSION TITLE
Lucky shot (GENERAL) [C]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#20] [Card Balance] [answered]
Lucky shot
Play during Combat. Your WARRIOR gains +3 to [S] during this Combat.

Artillery support
Warrior gains +5 to [S] during this COMBAT.

:arrow_right: add on "lucky shot" : ignore the bonus to [A]  for opponent's ARMOR.

**Comment by Sol-au-fein** [comment]
If it would ignore Subdermal Plates or Lifeguard armor it would be interesting card to play.

**Reply by 21-ric** [reply]
Because it's a "lucky shot" you can ignore the bonus for :
-carcass suit +1 to [A]
-combat armor +2 to [A]
-composite armor +2 to [A] or +3 [A] if it wounded
-subdermal plates base [A] is doubled

**Reply by Sol-au-fein** [reply]
Not lifeguard armor or Bodyarmor? 
Not much people are using common +2 armors. 
This might change in future though.
"Armor has no effect" will make it truly useful.

**Comment by TomekPL-1** [comment, marked as answer]
Good idea for Lucky shot 21-ric. I agree with Sol-au-fein that Lucky shot could have this ability:

Lucky Shot
Play during Combat. Your WARRIOR gains +3 to [S] and oponent ARMOR has no effect during this Combat.

**Reply by 21-ric** [reply]
I agree if it's not too powerfull. 

Perhaps add"may only be played during shoot combat".😊

**Reply by Sol-au-fein** [reply]
So we have our "critical hit" card.

**Comment by 21-ric** [comment]
Thanks to "lucky shot" we talk about armor.

What do you think if we add +2 to [A] for "armor of the true assassin"  and +1 to [A] for "body armor"?

Original armor of the tue assassin : +4 to [A]
Original body armor : +3 to [A]

If you are agree I can start a new discussion for these cards.

**Reply by Sol-au-fein** [reply]
Yeah, discussion about armor in another thread

**Comment by framerate** [comment]
Going in as 

> Play during Combat. Your WARRIOR gains +3 to [S] and oponent ARMOR has no effect during this Combat.
